examples/README.md

```diff
@@ -6,6 +6,10 @@
   - Learn how to write an HTTP service using annotations.
   - See [Annotated services](https://armeria.dev/docs/server-annotated-service).
 
+- `annotated-http-service-kotlin`
```

Comment:
Does gitpod support Kotlin? If so, we could add a link to it like we did for `annotated-http-service`.

Reply:
Gitpod seems to support Kotlin. https://www.gitpod.io/docs/languages/kotlin/

Reply:
I'll add a link.